client/my-sites/checkout/checkout-thank-you/test/index.js

```diff
@@ -138,19 +171,18 @@ describe( 'CheckoutThankYou', () => {
 		} );
 
 		test( 'Should be there for AT', () => {
-			const comp = shallow( <CheckoutThankYou { ...props } transferComplete={ true } /> );
-			expect( comp.find( 'component--AtomicStoreThankYouCard' ) ).toHaveLength( 1 );
+			render( <CheckoutThankYou { ...props } transferComplete={ true } /> );
+			expect( screen.queryByTestId( 'AtomicStoreThankYouCard' ) ).toBeVisible();
 		} );
 
 		test( 'Should not be there for AT', () => {
-			let comp;
-			comp = shallow( <CheckoutThankYou { ...props } transferComplete={ false } /> );
-			expect( comp.find( 'component--AtomicStoreThankYouCard' ) ).toHaveLength( 0 );
+			const { rerender } = render( <CheckoutThankYou { ...props } transferComplete={ false } /> );
+			expect( screen.queryByTestId( 'AtomicStoreThankYouCard' ) ).not.toBeInTheDocument();
 
 			isDotComPlan.mockImplementation( () => true );
 
-			comp = shallow( <CheckoutThankYou { ...props } /> );
-			expect( comp.find( 'component--AtomicStoreThankYouCard' ) ).toHaveLength( 0 );
+			rerender( <CheckoutThankYou { ...props } /> );
+			expect( screen.queryByTestId( 'AtomicStoreThankYouCard' ) ).not.toBeInTheDocument( 0 );
```

Comment:
Seems like we could omit the extra args here.

```suggestion
			expect( screen.queryByTestId( 'AtomicStoreThankYouCard' ) ).not.toBeInTheDocument();
```